README.md

```diff
@@ -1,6 +1,12 @@
 # Task list pattern - plugin for the GOV.UK Prototype Kit
 
-This plugin provides the code to use the GOV.UK Task list pattern in prototypes
+This plugin is no longer maintained.
+
+If you are using GOV.UK Prototype Kit version 5.0.0 or above, you should use the [task list component](https://design-system.service.gov.uk/components/task-list) from the GOV.UK Design System instead.
```

Comment:
Presumably this should be GOV.UK Frontend version 5.0.0, rather than Prototype Kit?

Reply:
@querkmachine opps yes, well spotted. I was still stuck with the mental model of the prototype kit getting released alongside govuk-frontend rather than being independent.